<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>644</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">/* =======================================================
   1. [필수] 전체 배경 설정 (아이보리: #FDFBF7)
   ======================================================= */
/* 중요: Widget 기반 폼에서 배경색이 적용되도록 수정했습니다. */
QWidget#Form, QWidget#centralwidget {
    background-color: #FDFBF7;
}

/* 메인 프레임도 배경색을 강제로 맞춰줍니다. */
QFrame#frame_main {
    background-color: #FDFBF7;
}

/* 기본 글꼴 및 색상 */
QLabel {
    color: #333333; 
    font-family: 'Segoe UI', 'Malgun Gothic', sans-serif;
    font-size: 14px;
}

/* =======================================================
   2. 사이드바 스타일 (배경색 구분 &amp; 투명 버튼)
   ======================================================= */
QFrame#frame_sidebar {
    background-color: #F0EFE9; /* 사이드바는 조금 더 진한 베이지 */
    border-right: 1px solid #D1D1D1;
}

/* 사이드바에 있는 메뉴 버튼들은 '투명'하게 설정 */
QPushButton {
    background-color: transparent;
    color: #333333;
    border: none;
    padding: 15px;
    text-align: left;
    font-size: 14px;
    font-weight: bold;
}

/* 마우스 올렸을 때 */
QPushButton:hover {
    background-color: #E6E3DA;
    color: #000000;
}

/* 선택되었을 때 (Checkable=True) */
QPushButton:checked {
    background-color: #E6E3DA;
    color: #2c3e50;
    border-left: 5px solid #2c3e50;
}

/* =======================================================
   3. 제목 라벨 스타일 (제목만 깔끔한 흰색 박스로!)
   ======================================================= */
/* Press, RF Pwr 등 제목 글자가 있는 라벨에만 흰색 배경을 줍니다. */
QLabel#label, QLabel#label_2, QLabel#label_3, QLabel#label_7, 
QLabel#label_8, QLabel#label_13, QLabel#label_14, 
QLabel#label_15, QLabel#label_16, QLabel#label_17, QLabel#label_18,
QLabel#label_19, QLabel#label_20, QLabel#label_21, 
QLabel#label_4, QLabel#label_5, QLabel#label_6 {
    
    background-color: #FFFFFF;  /* 흰색 배경 */
    color: #333333;             /* 진한 회색 글자 */
    
    border: 1px solid #DDDDDD;  /* 연한 테두리 */
    border-radius: 6px;         /* 둥근 모서리 */
    padding: 8px 12px;          /* 안쪽 여백 */
    
    font-weight: bold;
    font-size: 14px;
    min-height: 24px;
}

/* =======================================================
   5. [중요] 통계 버튼 스타일 복구 (여기가 핵심!)
   ======================================================= */
/* 위에서 '모든 버튼은 투명하라'고 했기 때문에, 
   이 버튼만 따로 '배경색과 테두리를 가져라'고 지정해줘야 합니다. 
   
   ★ 주의: Qt Designer에서 이 버튼의 objectName을 'btn_stats_open'으로 꼭 맞춰주세요! ★
*/

QPushButton#btn_stats_open {
    border: 1px solid #BBBBBB;   /* 회색 테두리 다시 생성 */
    background-color: #FFFFFF;   /* 흰색 배경 */
    border-radius: 6px;
    padding: 8px 15px;
    text-align: center;          /* 글자 가운데 정렬 */
    color: #333333;
    font-weight: bold;
}

/* 통계 버튼 마우스 올렸을 때 */
QPushButton#btn_stats_open:hover {
    background-color: #E6E3DA;   
    border: 1px solid #999999;
    color: #000000;
}

/* 통계 버튼 눌렀을 때 */
QPushButton#btn_stats_open:pressed {
    background-color: #D0D0D0;
    padding-top: 10px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame_4">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>공정 결함 통계 요약</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="mainf">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,7,1">
      <item>
       <widget class="QFrame" name="frame_5">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QComboBox" name="comboVariable"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_6">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,5,1">
         <item>
          <widget class="QFrame" name="frame_7">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5" stretch="0,3">
            <item>
             <widget class="QFrame" name="frame_11">
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_8">
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>변수 통계</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="histChart" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_8">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_6" stretch="0,3">
            <item>
             <widget class="QFrame" name="frame_12">
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_9">
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>이상 탐지</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="trendChart" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_9">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_7" stretch="0">
            <item>
             <widget class="QFrame" name="frame_2">
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_10" stretch="1,1,2">
               <item>
                <widget class="QFrame" name="frame_15">
                 <property name="frameShape">
                  <enum>QFrame::NoFrame</enum>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_11" stretch="1,3">
                  <item>
                   <widget class="QFrame" name="frame_16">
                    <property name="frameShape">
                     <enum>QFrame::NoFrame</enum>
                    </property>
                    <layout class="QVBoxLayout" name="verticalLayout_13">
                     <item>
                      <widget class="QLabel" name="label_4">
                       <property name="text">
                        <string>탐지 결과</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </item>
                  <item>
                   <widget class="QFrame" name="frame_10">
                    <property name="frameShape">
                     <enum>QFrame::NoFrame</enum>
                    </property>
                    <layout class="QVBoxLayout" name="verticalLayout_17">
                     <item>
                      <widget class="QLabel" name="lblCPK">
                       <property name="text">
                        <string>TextLabel</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QFrame" name="frame">
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_20">
                  <item>
                   <widget class="QFrame" name="frame_17">
                    <property name="frameShape">
                     <enum>QFrame::NoFrame</enum>
                    </property>
                    <layout class="QVBoxLayout" name="verticalLayout_19">
                     <item>
                      <widget class="QLabel" name="label_6">
                       <property name="text">
                        <string>CPK</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </item>
                  <item>
                   <widget class="QFrame" name="frame_13">
                    <property name="frameShape">
                     <enum>QFrame::NoFrame</enum>
                    </property>
                    <layout class="QVBoxLayout" name="verticalLayout_18">
                     <item>
                      <widget class="QLabel" name="cpk">
                       <property name="text">
                        <string>TextLabel</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QFrame" name="frame_14">
                 <property name="frameShape">
                  <enum>QFrame::NoFrame</enum>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_12" stretch="1,3">
                  <item>
                   <widget class="QFrame" name="frame_18">
                    <property name="frameShape">
                     <enum>QFrame::NoFrame</enum>
                    </property>
                    <layout class="QVBoxLayout" name="verticalLayout_14">
                     <item>
                      <widget class="QLabel" name="label_5">
                       <property name="text">
                        <string>TOP 10</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </item>
                  <item>
                   <widget class="QFrame" name="frame_19">
                    <property name="frameShape">
                     <enum>QFrame::NoFrame</enum>
                    </property>
                    <layout class="QVBoxLayout" name="verticalLayout_16">
                     <item>
                      <widget class="QListWidget" name="listTop10">
                       <property name="minimumSize">
                        <size>
                         <width>100</width>
                         <height>100</height>
                        </size>
                       </property>
                       <property name="maximumSize">
                        <size>
                         <width>256</width>
                         <height>16777215</height>
                        </size>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="statssummary">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_15">
         <item>
          <widget class="QTableWidget" name="txtStats">
           <property name="styleSheet">
            <string notr="true">background-color: #D0D0D0;  /* 불투명 회색 */
border-radius: 6px;
</string>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="mathbtn">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>수치형 기술 통계</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
